padertorch/train/trainer.py

```diff
@@ -248,7 +249,16 @@ def train(
         # Change model to train mode (e.g. activate dropout)
         self.model.train()
 
-        self.to(device)
+        if isinstance(device, (tuple, list)) \
+                and all([isinstance(d, int) for d in device]):
+            # multiple devises e.g. [0, 1], [0, 1, 2, 3], ...
```

Comment:
the second if statement should be an assert in my opinion

Reply:
Yes, an assert produces a better exception msg. I will change it

Reply:
done